### PR TITLE
Two adjustments of coding habits and  some  testInput instances  change  in conversion test

### PR DESCRIPTION
**Pull request by 20202200**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Two adjustments of coding habits and  some  testInput instances change  As follows:
1:  remove two repeated testInput instances
// Both are provided ,which is same as // Both are provided 4,6 
// list only ,which is same as // list only 4,6

2: add  four testInput  instances
// Both are provided - 4
// Both are provided - 6
// list only - 4
// list only - 6
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Comment by justinsb** [comment]
/lgtm

Looks like it needs a rebase to get rid of the merge commits though

File: pkg/apis/core/v1/conversion_test.go [hunk 1]
```diff
@@ -638,15 +638,6 @@ func Test_v1_NodeSpec_to_core_NodeSpec(t *testing.T) {
 		{
 			PodCIDR: "ace:cab:deca::/8",
 		},
-		// Both are provided
-		{
-			PodCIDR:  "10.0.1.0/24",
-			PodCIDRs: []string{"10.0.1.0/24", "ace:cab:deca::/8"},
```
Comment (justinsb) [inline pkg/apis/core/v1/conversion_test.go:644]: Maybe this was supposed to be PodCIDRs: []string{"10.0.1.0/24"} ?

File: pkg/apis/core/v1/conversion_test.go [hunk 2]
```diff
@@ -638,15 +638,6 @@ func Test_v1_NodeSpec_to_core_NodeSpec(t *testing.T) {
...
-		},
-		// list only
-		{
-			PodCIDRs: []string{"10.0.1.0/24", "ace:cab:deca::/8"},
```
Comment (justinsb) [inline pkg/apis/core/v1/conversion_test.go:648]: And similarly, maybe this was supposed to be PodCIDRs: []string{"10.0.1.0/24"} ?

File: pkg/apis/core/v1/conversion_test.go (hunk 1 above)
Comment (20202200) [reply pkg/apis/core/v1/conversion_test.go:644]: Thanks , Changed it

File: pkg/apis/core/v1/conversion_test.go (hunk 2 above)
Comment (20202200) [reply pkg/apis/core/v1/conversion_test.go:648]: Thanks , also Changed.

File: pkg/apis/core/v1/conversion_test.go [hunk 3]
```diff
@@ -657,6 +658,14 @@ func Test_v1_NodeSpec_to_core_NodeSpec(t *testing.T) {
 			PodCIDR:  "ace:cab:deca::/8",
 			PodCIDRs: []string{"ace:cab:deca::/8", "10.0.1.0/24"},
 		},
+		// list only - 4
+		{
```
Comment (20202200) [inline pkg/apis/core/v1/conversion_test.go:662]: https://github.com/kubernetes/kubernetes/blob/e1aa8197eddd277fb82be0b8bbc2ba2b4ca67af7/staging/src/k8s.io/api/core/v1/types.go#L5797
Should it be deleted when only PodCIDRS and PodCIDR not set? I can't make up my mind!

Comment (20202200) [reply pkg/apis/core/v1/conversion_test.go:662]: local test passed

**Pull request closed without merging**